packages/application-shell/src/components/locale-switcher/locale-switcher.tsx

```diff
@@ -136,7 +138,7 @@ const LocaleSwitcher = (props: Props) => {
           intlMessage={{
             ...messages.dialogLocaleDescription,
             values: {
-              newline: getNewLine,
+              newline: getNewLine as unknown as ReactNode,
```

Comment:
If we convert the `getNewLine` to a component (which it already is) we wouldn't need to to this double casting inline.